src/content/docs/accounts/accounts-billing/account-setup/student-program.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: New Relic for Students 
+metaDescription: Sign up for a New Relic student account 
+freshnessValidatedDate: never
+---
+
+Are you a student who is trying to learn about gathering and analyzing hardware and software telemetry? You can do this with New Relic for Students. This is a program that offers you free access to data, users, and New Relic features&mdash;even more options than you'd get from the Free Edition. 
```

Comment:
"who is" --> "who's"

Reply:
Is there some more meaningful way we can refer to "Free Edition"? I'm not sure it's reasonable to expect students will know what "free edition" is

Reply:
@paperclypse how about if we just don't try to compare this program with others (and leave this out)?

Reply:
@rhetoric101 works for me!